ProjFS.Mac/PrjFSKextTests/KauthHandlerTests.mm

```diff
@@ -1,3 +1,4 @@
+#include "../PrjFSKext/kernel-header-wrappers/kauth.h"
```

Comment:
Out of curiosity, why did this header need to get added if nothing else in `KauthHandlerTests.mm` changed?

Reply:
Nice Catch, the change was unnecessary here!

Reply:
I'm pretty sure we needed it because the enum declaration from `MockProc.hpp` for `KAUTH_RESULT_ALLOW` and such is no longer around. I'm not sure `#include "../PrjFSKext/kernel-header-wrappers/kauth.h"` is appropriate inside `MockProc.hpp` as nothing there requires any of the stuff in that header.